src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -0,0 +1,205 @@
+    return t('Integration Dashboard');
+  }
+
+  renderInstallData() {
+    const {total_uninstalls, total_installs} = this.state.stats;
+    return (
+      <React.Fragment>
+        <h5>{t('Installation Data')}</h5>
+        <Row>
+          <StatsSection>
+            <StatsHeader>{t('Total installs')}</StatsHeader>
+            <p>{total_installs}</p>
+          </StatsSection>
+          <StatsSection>
+            <StatsHeader>{t('Total uninstalls')}</StatsHeader>
+            <p>{total_uninstalls}</p>
+          </StatsSection>
+        </Row>
+        {this.renderInstallCharts()}
+      </React.Fragment>
+    );
+  }
+
+  renderTooltip(point, _pointIdx, chart) {
+    const timeLabel = chart.getTimeLabel(point);
+    const [installed, uninstalled] = point.y;
+
+    return (
+      <div style={{width: '150px'}}>
+        <div className="time-label">{timeLabel}</div>
+        <div className="value-label">
+          {intcomma(installed)} {`install${installed !== 1 ? 's' : ''}`}
+          {uninstalled > 0 && (
+            <React.Fragment>
+              <br />
+              {intcomma(uninstalled)} {`uninstall${uninstalled !== 1 ? 's' : ''}`}
+            </React.Fragment>
+          )}
+        </div>
+      </div>
+    );
+  }
+
+  renderInstallCharts() {
+    const {install_stats, uninstall_stats} = this.state.stats;
+    const points = install_stats.map((point, idx) => ({
+      x: point[0],
+      y: [point[1], uninstall_stats[idx][1]],
+    }));
+
+    return (
+      <Panel>
+        <PanelHeader>{t('Installations/Uninstallations over Time')}</PanelHeader>
+
+        <StackedBarChart
+          points={points}
+          height={150}
+          className="standard-barchart b-a-0 m-b-0"
+          barClasses={['accepted', 'rate-limited']}
+          minHeights={[2, 0, 0]}
+          gap={0.25}
+          tooltip={this.renderTooltip}
+        />
+      </Panel>
+    );
+  }
+
+  renderErrorLog() {
+    const {errors} = this.state;
+    return (
+      <React.Fragment>
+        <h5>{t('Error Log')}</h5>
+        <Panel>
+          <PanelHeader>
+            <TableLayout>
+              <div>{t('Time')}</div>
+              <div>{t('Organization')}</div>
+              <div>{t('Event Type')}</div>
+              <div>{t('Webhook URL')}</div>
+              <div>{t('Response Body')}</div>
+              <div>{t('Status Code')}</div>
+            </TableLayout>
+          </PanelHeader>
+
+          <PanelBody>
+            {errors.length > 0 ? (
+              errors.map((error, idx) => (
+                <PanelItem key={idx}>
+                  <TableLayout>
+                    <DateTime date={error.date} />
+                    <div>{error.organization.name}</div>
+                    <div>{error.eventType}</div>
+                    <OverflowBox>{error.webhookUrl}</OverflowBox>
+                    <div>{error.response.body}</div>
+                    <div>{error.response.statusCode}</div>
+                  </TableLayout>
```

Comment:
@nolachen Can we get the `statusCode` further left because it's a relatively important field?